data/human-analytica.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "Human Analytica",
+  "url": "https://humananalytica.it",
+  "type": "Consulting",
+  "categories": ["marketing_writing"],
+  "remote_policy": "Hybrid",
+  "hiring_policy": "Direct"
+}
```

Comment:
Hi @valentinocossar, `career_page_url` is missing, can you please add one? (if the company does not have a career page, a LinkedIn job section works too)

Reply:
Hi @edoardocostantinidev, missing career_page_url added, thanks!